app-creator/src/widgets/ui-map/ui-map.ts

```diff
@@ -0,0 +1,48 @@
+import {} from 'googlemaps';
+import { customElement, html, LitElement } from 'lit-element';
+
+@customElement('ui-map')
+export class Map extends LitElement {
+  /**
+   * Sets zoom property on map.
+   */
+  zoom = 4;
```

Comment:
It might be a good idea to make these properties private and expose getters and setters instead, which will actually set the map property (map.setZoom etc.) when the property is changed.